pkg/components/aws-ebs-csi-driver/component.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*Package awsebscsidriver - The golang plumbing for the aws-ebs-csi-driver
+component */
+package awsebscsidriver
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/hcl/v2"
+	"github.com/hashicorp/hcl/v2/gohcl"
+	"github.com/kinvolk/lokomotive/pkg/components"
+	"github.com/kinvolk/lokomotive/pkg/components/util"
+)
+
+const name = "aws-ebs-csi-driver"
+
+const chartValuesTmpl = `
+{{- if .EnableDefaultStorageClass }}
```

Comment:
If you set the property to `false`, the value never renders, to it's not possible to disable it.